seed/models/tax_lots.py

```diff
@@ -488,4 +488,26 @@ class Meta:
         index_together = [['state', 'name'], ['parent_state1', 'parent_state2']]
 
 
+@receiver(pre_save, sender=TaxLotState)
+def sync_latitude_longitude_and_long_lat(sender, instance, **kwargs):
+    try:
+        original_obj = sender.objects.get(pk=instance.pk)
+    except sender.DoesNotExist:
+        pass  # Occurs on object creation
+    else:
+        # Sync Latitude, Longitude, and long_lat fields if applicable
+        latitude_change = original_obj.latitude != instance.latitude
+        longitude_change = original_obj.longitude != instance.longitude
+        long_lat_change = original_obj.long_lat != instance.long_lat
+        lat_and_long_both_populated = instance.latitude is not None and instance.longitude is not None
+
+        # The 'not long_lat_change' condition removes the case when long_lat is changed by an external API
+        if (latitude_change or longitude_change) and lat_and_long_both_populated and not long_lat_change:
```

Comment:
hmm. i get this logic, but I wonder if there is an easier way. Approving for now.